sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/cra-response-integration.scheduler.ts

```diff
@@ -24,9 +24,7 @@ export class CRAResponseIntegrationScheduler extends BaseScheduler<void> {
    * @returns Summary with what was processed and the list of all errors, if any.
    */
   @Process()
-  async processResponses(
-    job: Job<void>,
-  ): Promise<ProcessResponseQueueOutDTO[]> {
+  async processResponses(job: Job<void>): Promise<ProcessResponseQueue[]> {
```

Comment:
Optional comment. `const summary = new QueueProcessSummary({
      appLogger: this.logger,
      jobLogger: job,
    });`
and `summary.info` instead of `this.logger.log`

Reply:
CRA is not part of this PR

Reply:
That is why optional 😊